Removed missing reference.

moorepants opened the pull request:
This is causing a compile error locally for me but didn't see to cause issues on travis. Just removing for now so it compiles.

moorepants committed:
Removed missing reference.

moorepants commented:
@Upabjojr

Upabjojr commented:
Maybe this link as a citation or footnote:

https://reference.wolfram.com/language/tutorial/TheSoftwareEngineeringOfTheWolframSystem.html

moorepants committed:
Added citation to the Wolfram docs.

It would be better if we could cite a white paper that states this information
for posterity.

moorepants commented:
I see you used this commit, so I'll just merge.
